src/canvas.class.js

```diff
@@ -408,6 +408,13 @@
       return objsToRender;
     },
 
+    isErasing: function () {
+      return this.isDrawingMode &&
+        this.freeDrawingBrush &&
+        this.freeDrawingBrush.type === 'eraser' &&
+        this.freeDrawingBrush._isErasing;
+    },
```

Comment:
can you extend this in the brush file ? So we do not add functionalities to the canvas that aren't used by default.

Reply:
DONE